CHANGELOG.md

```diff
@@ -1,12 +1,24 @@
+## [3.3.9] - TBD
```

Comment:
This change doesn't seem to belong to this PR.

Reply:
I corrected the old changelog and added this pr as well
The point is to have it up to date and update it on every change after this one